snuba/settings/__init__.py

```diff
@@ -169,7 +170,7 @@
 COLUMN_SPLIT_MAX_RESULTS = 5000
 
 # Migrations in skipped groups will not be run
-SKIPPED_MIGRATION_GROUPS: Set[str] = {"querylog", "profiles", "replays"}
+SKIPPED_MIGRATION_GROUPS: Set[str] = {"querylog", "profiles", "functions", "replays"}
```

Comment:
Now this setting is also overridden in the production config.
Please remember to send a PR to add your group there.

Reply:
where can i find this production config?

Reply:
https://github.com/getsentry/ops/pull/4902